src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -968,3 +971,60 @@
 type: command
 short-summary: Update metadata for an IoT Central application.
 """
+
+helps['iot hub private-endpoint-connection approve'] = """
+type: command
+short-summary: Approve a private endpoint connection to an IoT Hub.
+examples:
+  - name: Approve a private endpoint connection to an IoT Hub by name
+    text: >
+        az iot hub private-endpoint-connection approve -n MyPrivateEndpointConnection --hub-name MyIotHub -g MyGroup --desc 'Approved connection'
+  - name: Approve a private endpoint connection to an IoT Hub by ID
+    text: >
+        az iot hub private-endpoint-connection approve --id {id} --desc 'Approved connection'
+"""
+
+helps['iot hub private-endpoint-connection reject'] = """
+type: command
+short-summary: Reject a private endpoint connection to an IoT Hub.
+examples:
+  - name: Reject a private endpoint connection to an IoT Hub by name
+    text: >
+        az iot hub private-endpoint-connection reject -n MyPrivateEndpointConnection --hub-name MyIotHub -g MyGroup --desc 'Rejected connection'
+  - name: Reject a private endpoint connection to an IoT Hub by ID
+    text: >
+        az iot hub private-endpoint-connection reject --id {id} --desc 'Rejected connection'
```

Comment:
Recommend changing the  `--desc` short alias for description to something else. It's too close to `descending` as in `desc` is often used for descending in different tools.